tools/clang/unittests/HLSLExec/ShaderOpTest.cpp

```diff
@@ -1067,6 +1067,7 @@ void ShaderOpTest::RunShaderOp(ShaderOp *pShaderOp) {
   CreateCommandList();
   RunCommandList();
   CopyBackResources();
+  CloseHandle(m_hFence);
```

Comment:
Should initialize this in the constructor (or where it's declared in the header file).